.github/workflows/coverage.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       uses: actions-rs/cargo@v1
       with:
         command: tarpaulin
-        args: --workspace
+        args: --out Xml
```

Comment:
It's my fault, I've removed it by accident in https://github.com/matrix-org/matrix-rust-sdk/pull/753. Sorry for that.

Reply:
It happens, also I approved the PR... Besides, we're still not sure that this fixes the problem.

Reply:
Seems to be fixed indeed.